de_templates/ingest/Ingest_SourceType_SourceName_DQ/spark_jobs/ingest_dq.py.jinja

```diff
@@ -3,6 +3,9 @@ from pysparkle.pysparkle_cli import cli
 
 
 def call_pysparkle_entrypoint():
+    """
```

Comment:
General remark about jinja templates: when I tested them today, there were many more issues with black in the output files - do we want to handle these problems here? I guess jinja is not getting picked up by the current formatters. Here's a list of files that are problematic after I generate a DQ pipeline:

<img width="981" alt="image" src="https://github.com/amido/stacks-azure-data/assets/131348498/5e35c7fe-5463-40be-a2e2-b54eeeba5495">

Reply:
I will check these

Reply:
This should be address by the commit https://github.com/amido/stacks-azure-data/pull/131/commits/814e718839de64361fa1ebbeeac2a8b0f1306945

Reply:
looks good, thanks!